command/ssh_test.go

```diff
@@ -65,6 +76,17 @@ func TestParseSSHCommand(t *testing.T) {
 			"",
 			nil,
 		},
+		{
+			"Parse the username out of -o User=\"User Name\" #7912",
+			[]string{
```

Comment:
Might be a good idea to also provide an invalid test case to verify that we handle it correctly, e.g. `"-o", "User=\"username"` (no closing quote).